packages/commonwealth/client/scripts/views/pages/discussions/threads_overview.tsx

```diff
@@ -0,0 +1,65 @@
+/* @jsx m */
+
+import m from 'mithril';
+
+import 'pages/discussions/threads_overview.scss';
+
+import app from 'state';
+import { LoadingRow } from '../../components/loading_row';
+import { CWText } from '../../components/component_kit/cw_text';
+import { CWButton } from '../../components/component_kit/cw_button';
+import { ThreadsOverviewTopicSummaryRow } from './threads_overview_topic_summary_row';
+
+export class ThreadsOverview implements m.ClassComponent {
+  private initializing: boolean;
+
+  oninit() {
+    app.recentActivity.getRecentTopicActivity().then(() => {
+      m.redraw();
+    });
+  }
+
+  view() {
+    if (this.initializing) {
+      return LoadingRow;
+    }
+
+    const topics = app.topics.getByCommunity(app.activeChainId());
+
+    return (
+      <div class="ThreadsOverview">
+        <div class="header-row">
+          <CWText
+            type="h5"
+            fontWeight="semiBold"
+            className="threads-header-row-text"
+          >
+            Topic
+          </CWText>
+          <div class="threads-header-container">
+            <CWText
+              type="h5"
+              fontWeight="semiBold"
+              className="threads-header-row-text"
+            >
+              Recent threads
+            </CWText>
+            {/* <CWButton buttonType="mini" label="Create Thread" iconName="plus" /> */}
+          </div>
+        </div>
+        {topics.map((topic) => {
+          const monthlyThreads = app.threads.overviewStore
```

Comment:
From designs, it looks like pinned threads should always be up top, even over more recently active communities. This may also imply that pinning alone is grounds for inclusion on this summary page, recent activity be damned. 

<img width="849" alt="image" src="https://user-images.githubusercontent.com/22281010/194388892-e691d4ab-a043-4b62-a809-12d48f82ce5e.png">

Reply:
We'll fix this later